README.md

```diff
@@ -63,69 +63,69 @@ cargo build
 
```

Comment:
```suggestion
To run the unittest tests locally, you need to start a local postgres instance, then run the tests.  You can do this at the mob prompt:

```
# From the root of the cloned repository
./mob prompt

# Start postgres
service postgresql start

# Run the tests
cargo test
```
```

Reply:
hmm this did not do what I expected....It seems like a couple limitations of Github: you can't make a comment on a part of the commit that wasn't changed, and you also cant nest a code block within a suggestion.  Basically, I was suggesting the following raw text go right *before* the `## Overview line` (as the last paragraph of Build Instructions section)...


To run the unittest tests locally, you need to start a local postgres instance, then run the tests.  You can do this at the mob prompt:

```
# From the root of the cloned repository
./mob prompt

# Start postgres
service postgresql start

# Run the tests
cargo test
```

Reply:
Done! Thank you!